sci-biology/nistats/nistats-9999.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 multilib git-r3 flag-o-matic
+
+DESCRIPTION="Neuroimaging tools for Python"
+HOMEPAGE="http://nipy.org/"
+SRC_URI=""
+EGIT_REPO_URI="https://github.com/nistats/${PN}"
+
+LICENSE="BSD"
+SLOT="0"
+IUSE=""
+KEYWORDS=""
+
+DEPEND="
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	dev-python/numpy[${PYTHON_USEDEP}]"
+RDEPEND="${DEPEND}
```

Comment:
I thought it would be shorter than making a separate COMMONDEPEND variable just to hold numpy.